lib/jwt/jwt-authenticator.js

```diff
@@ -29,7 +29,7 @@ JwtAuthenticator.prototype.withCookie = function withCookie(cookieName){
 };
 
 JwtAuthenticator.prototype.unauthenticated = function unauthenticated(){
-  return new ApiAuthRequestError('Unauthorized', 401);
+  return new ApiAuthRequestError({userMessage:'Unauthorized', statusCode: 401});
```

Comment:
This seems fairly repeated. I'd probably make it an own error that was extended from ApiAuthRequestError. Something like `ApiUnauthorizedError` and have it set the statusCode to `401` and default userMessage to `Unauthorized`. Then add an optional userMessage parameter, so that if it is specified, then it will use that instead. What do you think?